src/test/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTaskTest.java

```diff
@@ -17,10 +18,7 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Paths;
 import java.time.Instant;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
```

Comment:
Please run formatter and re-organize imports , use of wildcard imports is not recommended.

Reply:
ntelliJ IDEA uses wildcard imports (import java.util.*;) whenever the number of classes imported from the same package reaches the default 5 limit. I will disable the same and will checkin the with individual imports.

Reply:
Got it! , but you get the overall idea. May be we should use the same formatter we have in the JAVA SDK for imports so that it runs with the import lifecycle.